### PR TITLE
Bump cpp_demangle to 0.4.

### PR DESCRIPTION
**Pull request by khuey**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -50,7 +50,7 @@ split-debuginfo = 'packed'
 
 [features]
 default = ["rustc-demangle", "cpp_demangle", "std-object", "fallible-iterator", "smallvec"]
-std = ["gimli/std"]
+std = ["gimli/std", "cpp_demangle/std"]
```
Comment (philipc) [inline Cargo.toml:53]: This doesn't appear to be needed, and forces `cpp_demangle` enabled.

Comment (khuey) [reply Cargo.toml:53]: Hmm, it appears that `addr2line` just tosses errors from `cpp_demangle`, so yeah, that seems true.

**Comment by khuey** [comment]
Poke.

**Review by philipc — APPROVED** [review]

**Comment by philipc** [comment]
Looks fine. I still need to find a way to fix #256 before CI will pass. I normally do this sort of change as part of the release.

**Pull request merged**